tests/unittests/dataset_tests.py

```
@@ -1148,6 +1148,89 @@
+        #
+        # Main process
+        #
+
+        dataset.update_samples(
+            update_fcn, workers=1, parallelize_method="process"
+        )
+
+        self.assertTupleEqual(dataset.bounds("int"), (3, 52))
+
+    @drop_datasets
+    def test_map_samples(self):
```

Ruff (0.8.2)

Redefinition of unused `test_map_samples` from line 1113

(F811)